content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -105,6 +105,10 @@ to use the last chunk of container log output if the termination message file
 is empty and the container exited with an error. The log output is limited to
 2048 bytes or 80 lines, whichever is smaller.
```

Comment:
2048 bytes or 4096 bytes? Which is it?

Reply:
4096 bytes --> terminationMessagePath

2048 bytes --> terminationMessagePolicy